tests/functional/u/useless/useless_super_delegation_py3.py

```diff
@@ -41,3 +41,30 @@ def __init__(self, thing: object) -> None: # [useless-super-delegation]
 class Test:
     def __init__(self, _arg: List[int]) -> None:
         super().__init__()
+
+
+import random
+
+class ReturnTypeAny:
+    choices = ['a', 1, (2, 3)]
+
+    def draw(self) -> None:
```

Comment:
This annotation is incorrect, is that intentional?

Reply:
Well spotted, that wasn't intentional.

While I was fixing that, I realized there's another case we need to think about:

    import typing
    from typing import Any

    class Base:
        def foo(self) -> Any:
            pass

    class Derived:
        def foo(self) -> typing.Any:
            return super().foo()

This will be harder to fix, and it looks like the existing rules around type hints on parameters don't cope with this case either, so I suggest this can be handled as a later fix to both parameter type hints and return type hint.